samples/web/content/apprtc/apprtc_test.py

```diff
@@ -5,7 +5,9 @@
 
 import apprtc
 import constants
+import gcmrecord
 import json
+from google.appengine.datastore import datastore_stub_util
```

Comment:
I thought about it, but didn't because I was trying to make the file less long. They should probably be there though, I'll move them.